### ISSUE TITLE
v2 not working with Nomic

### ISSUE THREAD

**Issue by manueldalb2** [blockpane/tenderduty#31]
Seems that v2 (i'm using lastest v2.0.2) doesn't work with Nomic chain anymore

refreshing signing info for nomicvalcons1y83npk4q35f6xjav0l9qjgm6xxjmkmddwg5eh8 could not find validator nomicvalcons1y83npk4q35f6xjav0l9qjgm6xxjmkmddwg5eh8

address is retrieved using the HEX value of address on a RPC node

![image](https://user-images.githubusercontent.com/65422254/185078358-aaca73d2-7ef4-4323-863d-7302f622e67d.png)

and https://slowli.github.io/bech32-buffer/

![image](https://user-images.githubusercontent.com/65422254/185078394-1b96fc07-b873-497f-8483-8f3fd2446a6f.png)

this was a working configuration (reported by other users, not tried by me)

**Comment by Bosco-2019** [comment]
Having the same issue here, would be great to have Nomic alerting.

**Comment by dimiandre** [comment]
@blockpane do you need help on this?

**Comment by blockpane** [comment]
Here's what I can tell on this one.

Tenderduty is pulling validator info using a request to the `/abci_query` RPC endpoint with the `/cosmos.staking.v1beta1.Query/Validator` path. I do this so only the RPC is needed, and not RPC + REST|GRPC.

It doesn't look like protobuf/ABCI queries are supported in Nomic. There aren't a ton of pure ABCI queries used in tenderduty, but more than getting the consensus key is broken.

After looking at their javascript client, it appears there is a websocket endpoint for getting _all_ validators with the same info, but nothing for a specific validator. So it might be possible to pull this info and loop, but will still have issues with getting slashing params and other ABCI lookups.

**Comment by Bosco-2019** [comment]
Hey so just wanted to report that nomic is mostly working with the latest version using the valcons address. We won't get our missed block counter but it does alert when a node is down. 

![image](https://github.com/blockpane/tenderduty/assets/63794741/876c0643-0fb7-4856-9f65-b596c3dd41ab)

**Comment by StakeandRelax-Validator** [comment]
you are right, now it's correctly working! It does not resolve the moniker only, at least on my side
